src/meta/processors/admin/AdminClient.cpp

```diff
@@ -512,9 +512,14 @@ folly::Future<Status> AdminClient::createSnapshot(GraphSpaceID spaceId, const st
 
     folly::Promise<Status> pro;
     auto f = pro.getFuture();
+
+    /**
+     * Don't need retry.
+     * Because existing checkpoint directories leads to fail again.
+     **/
     getResponse(allHosts, 0, std::move(req), [] (auto client, auto request) {
         return client->future_createCheckpoint(request);
-    }, 0, std::move(pro), 1 /*The snapshot operation only needs to be retried twice*/);
+    }, 0, std::move(pro), 0);
```

Comment:
What if there are some connection failure?

Reply:
An error returned, and rerun the create command by console. Because some nodes may succeed, the checkpoint directory will exist, retry will not work.

Reply:
make sense.